core/statistics.c

```diff
@@ -326,16 +326,18 @@ void get_selected_dives_text(char *buffer, size_t size)
 
 #define SOME_GAS 5000 // 5bar drop in cylinder pressure makes cylinder used
 
-bool is_cylinder_used(struct dive *dive, int idx)
+bool is_cylinder_used(struct dive *dive, int idx, bool ignore_plannned)
```

Comment:
What do you prefer? Do you want me to rather add a function is_cylinder_used_in_non_planned_dives() or do it inline where it's used? My first version even had a goto... Please advise.